lang-java/openjdk-23/autobuild/defines.stage2

```diff
@@ -0,0 +1,14 @@
+PKGNAME=openjdk-23
+PKGSEC=java
+PKGDEP="ca-certs nss xdg-utils hicolor-icon-theme alsa-lib gtk-2 cups \
+        fontconfig unzip zip cpio giflib"
+BUILDDEP="openjdk-22"
+PKGDES="OpenJDK Java Runtime Environment (JRE), Java Development Environment (JDK), and IcedTea-Web"
```

Comment:
Please mark version in description - also, are you sure that IcedTea is included?